CHANGELOG.md

```diff
@@ -34,8 +35,7 @@
 - Fix bug in addon loading with namespaced packages @tiberiuichim
 - Japanese translation updated @terapyon
 
-
-- Upgrade razzle to `^3.1.5`. @tiberiuichim
+* Upgrade razzle to `^3.1.5`. @tiberiuichim
```

Comment:
@iFlameing I guess that was not intended. :)

Reply:
Yes!!! It is because of prettier.